src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -99,22 +99,11 @@ def get(self, svc_id):
         """
         Returns collected data about an OSD.
 
-        :return: Returns the requested data. The `histogram` key may contain a
-                 string with an error that occurred if the OSD is down.
+        :return: Returns the requested data.
         """
-        try:
-            histogram = CephService.send_command(
-                'osd', srv_spec=svc_id, prefix='perf histogram dump')
-        except SendCommandError as e:  # pragma: no cover - the handling is too obvious
-            if 'osd down' in str(e):  # pragma: no cover - no complexity there
-                histogram = str(e)
-            else:  # pragma: no cover - no complexity there
-                raise
-
         return {
             'osd_map': self.get_osd_map(svc_id),
             'osd_metadata': mgr.get_metadata('osd', svc_id),
-            'histogram': histogram,
```

Comment:
I don't think this is a good idea to remove this now that the Dashboard REST API is the official Ceph REST API. Maybe this should be discussed in the next Dashboard standup meeting.

Reply:
Thanks for pointing this out @votdev. As discussed in today's stand-up, we may get rid of this (also saving one `send_command()` call per OSD every 5 secs) and move it to a separate end-point (`api/osd/{id}/histogram`): https://tracker.ceph.com/issues/46898